### PR TITLE
Hibernate 1

### PR DESCRIPTION
**Pull request by IwalkInSilence**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,14 +19,21 @@
...
-            <artifactId>log4j-api</artifactId>
-            <version>2.14.0</version>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.22</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-java8</artifactId>
```
Comment (Artificial-Friend) [inline pom.xml:28]: we need to remove dependencies that we are not using

**Review by Artificial-Friend — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (IwalkInSilence) [reply pom.xml:28]: without this dependency all broken up

File: src/main/java/project/cinema/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert movie entity", e);
```
Comment (Sofasmile) [inline src/main/java/project/cinema/dao/impl/MovieDaoImpl.java]:
```suggestion
            throw new DataProcessingException("Can't insert movie entity " + movie, e);
```

**Review by Sofasmile — CHANGES_REQUESTED** [review]
good job)

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**